plugin/src/utils/remix_storage.ts

```diff
@@ -19,3 +19,25 @@ export const getAllContractFiles = async (remixClient: RemixClient, path: string
   }
   return files
 }
+
+export const getContractTargetPath = (allContracts: ContractFile[], contractFileName: string) => {
```

Comment:
Again same edge case:
```
/root
--/a
----Bank.sol
----/b
------Bank.sol
```

What shall be happening is kinda like that `currentFilePath.split(./).slice(0, -1).join('/')`

Reply:
Good suggestion. Done 👍